### ISSUE TITLE
Volt 0.8.26 wants to overwrite ruby 2.2.2's rake

### ISSUE THREAD

**Issue by mikestok** [voltrb/volt#142]
When installing `volt` with an `rbenv` installed `ruby` 2.2.2 it attempts to overwrite `rake`.  Should the version requirement in `gemfile.spec` be updated to allow for 10.4.x  versions of rake? 
```
07:18 $ ruby -v
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14]
✔ ~
07:19 $ rake -V
rake, version 10.4.2
✔ ~
07:19 $ gem install volt
[...]
Fetching: rake-10.0.4.gem (100%)
rake's executable "rake" conflicts with /Users/mike/.rbenv/versions/2.2.2/bin/rake
Overwrite the executable? [yN]
```

**Comment by mikestok** [comment]
ah, `volt-0.8.27.beta9` seems OK.

**Comment by ryanstout** [comment]
@mikestok yea, this is fixed in the 0.9.0 prerelease.
